deon/__main__.py

```diff
@@ -0,0 +1,4 @@
+from .cli import main
+
+if __name__ == "__main__":
+    main()
```

Comment:
The help text currently looks like this if you run it through `python -m`. The `__main__.py` isn't what we want. 

```
11:06:24 ❯ python -m deon --help
Usage: __main__.py [OPTIONS]

  Easily create an ethics checklist for your data science project.
```

It looks like click lets you easily specify what goes there, so you can change it to say `python -m deon`. See this example: https://github.com/pallets/click/issues/1399#issuecomment-532708528

Reply:
Done.